src/ixdat/techniques/ec_ms.py

```diff
@@ -6,3 +6,6 @@
 
 class ECMSMeasurement(ECMeasurement, MSMeasurement):
```

Comment:
Yes! Yes! Yes! Just tested and this inheritence works as intended. 
The ECMSMeasurement returned by your reader has both the methods `get_potential` from ECMeasurement and `get_calib_signal` and they both work as intended!